src/draw/lv_image_decoder.h

```diff
@@ -49,6 +49,22 @@ struct _lv_image_decoder_dsc_t;
 struct _lv_image_decoder_t;
 struct _lv_cache_entry_t;
 
+/**
+ * Image decoder args.
+ * It determines how to decoder an image, e.g. whether to premultiply the alpha or not.
+ * It should be passed to lv_img_decoder_open() function. If NULL is provided, default
+ * args are used.
+ *
+ * Default args:
+ * all field are zero or false.
+ */
+typedef struct _lv_image_decoder_args_t {
+    int32_t frame_id;       /*A not use args*/
+    bool stride_align;      /*Whether stride should be aligned*/
+    bool premultiply;       /*Whether image should be premultiplied or not after decoding*/
+    bool no_cache;          /*Whether this image should be kept out of cache*/
```

Comment:
What does it mean in the practice e.g. in case of a PNG image?

Reply:
For PNG decoders support output stride(assume so), it can use the `args` directly. Otherwise, a global helper function may be used to convert stride by 2d-copying to a new buffer.

Similar for `premultiply`.

`no_cache` is a use case for frame animation where every image could only use once. Or a RAM image from video, that updates every frame.

Reply:
Got it.